bin/registries/docker-pull.gcr.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+echo "Preping to pull image from GCR."
+
+if [ -z "$GCP_PROJECT" || -z "$GCP_CLUSTER" || -z "$GCP_ZONE" ]; then
+  echo "Missing GCP project, cluster, or zone!  Aborting"
+  exit 1
+fi
```

Comment:
Can you split these into separate conditionals so the error message is more specific? There may be cases where the environment is difficult to inspect and it would help if rok8s-scripts can tell us what it knows. 

Example case being that we don't have write access to a repo and can't look at the environment in Circle easily.